rllib/algorithms/appo/tf/appo_tf_learner.py

```diff
@@ -5,7 +5,7 @@
     AppoLearner,
     LEARNER_RESULTS_CURR_KL_COEFF_KEY,
     LEARNER_RESULTS_KL_KEY,
-    OLD_ACTION_DIST_KEY,
+    OLD_ACTION_DIST_LOGITS_KEY,
```

Comment:
We are not returning Distributions anymore from any forward methods, only dist inputs (logits).